src/components/EditRatingModal.jsx

```diff
@@ -0,0 +1,91 @@
+import Col from 'react-bootstrap/Col'
+import Form from 'react-bootstrap/Form'
+import Modal from 'react-bootstrap/Modal'
+import { Button } from 'react-bootstrap'
+import { useState } from 'react'
+import { rateItemAsync } from '../redux/items/thunks'
+import { useDispatch } from 'react-redux'
+
+import '../styles.css'
+
+const RATINGS = [
+  { id: '1', name: '1' },
+  { id: '2', name: '2' },
+  { id: '3', name: '3' },
+  { id: '4', name: '4' },
+  { id: '5', name: '5' }
+]
```

Comment:
is this component supposed to use the ratings constant?

Reply:
words are hard at 2am :')